index.bs

```diff
@@ -1405,6 +1424,18 @@ a [=string=] |key|, and a possibly null 64-bit unsigned integer |default|:
     error.
 1. Return |value|.
 
+To <dfn>parse an optional 32-bit unsigned integer</dfn> given a [=map=] |map|,
+a [=string=] |key|, and a possibly null 32-bit unsigned integer |default|:
+
+1. If |map|[|key|] does not [=map/exists|exist=], return |default|.
+1. If |map|[|key|] is not a [=string=], return an error.
```

Comment:
We don't currently require the use of strings when parsing other 32-bit unsigned integers, e.g. source-side [trigger data](https://wicg.github.io/attribution-reporting-api/#parse-trigger-data-into-a-trigger-spec-map) and [summary buckets](https://wicg.github.io/attribution-reporting-api/#parse-summary-buckets), so the same should apply here.

Reply:
Done.